components/Navbar.vue

```diff
@@ -219,7 +218,7 @@ export default class NavbarMenu extends mixins(
   private lastScrollPosition = 0
   private artistName = ''
   public isBurgerMenuOpened = false
-  private isMobile = window.innerWidth < 1024 ? true : isMobileDevice
```

Comment:
I look through the git history. This line seems to fix a bug on laptops. Will it introduce original bug after the change?
<img width="1092" alt="image" src="https://user-images.githubusercontent.com/31397967/211035722-9711486b-4b1b-45f9-87c5-f879189fe212.png">

Reply:
I don't know which bug you are referring to, but generally, we've moved from caring about the touch/"mobileness" of the device and moved to, strictly speaking, screen size differentiation.

Small screens are treated as mobiles when it comes to the navbar. I'm unsure if this is right from the UX perspective, but this was the requirement.